src/actions/assetsActions.js

```diff
@@ -551,61 +556,68 @@ export const fetchAssetsBalancesAction = (assets: Assets, showToastIfIncreased?:
 
 export const fetchInitialAssetsAction = () => {
   return async (dispatch: Dispatch, getState: () => Object, api: Object) => {
-    const { user: { data: { walletId } } } = getState();
+    const {
+      user: { data: { walletId } },
+      accounts: { data: accounts },
+    } = getState();
+
     dispatch({
       type: UPDATE_ASSETS_STATE,
       payload: FETCHING_INITIAL,
     });
     await delay(1000);
     const initialAssets = await api.fetchInitialAssets(walletId);
-
     if (!Object.keys(initialAssets).length) {
       dispatch({
         type: UPDATE_ASSETS_STATE,
         payload: FETCH_INITIAL_FAILED,
       });
       return;
     }
-
+    const activeAccountId = getActiveAccountAddress(accounts);
```

Comment:
```suggestion
    const activeAccountId = getActiveAccountId(accounts);
```

Reply:
Hm, both of those methods return the same thing - id prop of account object:
getActiveAccountId
`const getActiveAccountId = (accounts: Accounts): string => {
  const activeAccount = getActiveAccount(accounts);
  if (!activeAccount) {
    return '';
  }
  return activeAccount.id;
};`

getActiveAccountAddress:
`export const getActiveAccountAddress = (accounts: Accounts): string => {
  const activeAccount = getActiveAccount(accounts);
  if (!activeAccount) {
    return '';
  }
  return getAccountAddress(activeAccount);
};`

where getAccountAddress is:
`const getAccountAddress = (account: Account): string => {
  return account.id;
};`

Do we need them both? It raises some confusion

Reply:
The primary idea was to separate id from address as they supposed to be different. But now I see that the address as an id worked out pretty well and there is no need to 2 same things under one name. We should unify it